### PR TITLE
Added announcement bar feature enhancement

### PR DESCRIPTION
**Pull request by liamjosephsilk**
Used the docosaurus v2 built in announcement bar.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,29 +8,39 @@ module.exports = {
   projectName: "blitzjs.com", // Usually your repo name.
   themeConfig: {
     algolia: {
-      apiKey: 'c4db860ae4162be48d4c867e33edcaa2',
-      indexName: 'blitzjs',
+      apiKey: "c4db860ae4162be48d4c867e33edcaa2",
+      indexName: "blitzjs",
+    },
+    announcementBar: {
```
Comment (merelinguist) [inline docusaurus.config.js:14]: Thanks for this! Could you please:

1. Make the bar have the same background colour as the footer

2. Use white text for readability

Comment (liamjosephsilk) [reply docusaurus.config.js:14]: I'll make the changes, in local developement I changed some styles in the custom css file, however didn't realise classnames are converted on build so i'll check into this and fix the changes.

**Comment by liamjosephsilk** [comment]
I have added the changes from the review. I had to swizzle from the announcement bar in order to change some internal css to the component.

File: docusaurus.config.js [hunk 2]
```diff
@@ -8,29 +8,40 @@ module.exports = {
...
+      id: "progress",
+      content: "Site Still In Progress",
```
Comment (flybayer) [inline docusaurus.config.js:16]: Thank you @liamjosephsilk! Let's make this a bit more prominent 
```suggestion
      content: "🚧Excuse the mess — design in progress! 🚧",
```

**Pull request merged**